package.json

```diff
@@ -20,5 +23,26 @@
   "bugs": {
     "url": "https://github.com/umeshp7/postman-collection-code-generators/issues"
   },
-  "homepage": "https://github.com/umeshp7/postman-collection-code-generators#readme"
+  "homepage": "https://github.com/umeshp7/postman-collection-code-generators#readme",
+  "dependencies": {
+    "async": "^3.2.0",
+    "node-fetch": "^2.6.0",
+    "postman-code-generators": "git+https://github.com/someshkoli/postman-code-generators.git#sdkGenerator",
+    "postman-collection": "^3.6.2",
+    "shelljs": "^0.8.4"
```

Comment:
the root package.json should have only the dependencies required for the root functions. Rest should be inside the respective package.json of the sdkgens

Reply:
Is this regarding the node-fetch ? It is being used to get collection when collection url is provided.

Reply:
ok - remove the ones that are not required.